Cleanup status script

mchenryc opened the pull request:
While working updating to freeciv server, I found these changes useful.

Overall status will appear as:
```
vagrant@ubuntu-bionic:/vagrant/freeciv-web$ ../scripts/status-freeciv-web.sh
Checking that Freeciv-web is running correctly... (-v for verbose)

Checking nginx                                     OK!
Checking Tomcat                                    OK!
Checking freeciv-web on Tomcat                     OK!
Checking freeciv-proxy directly                    OK!
Checking freeciv-proxy through nginx               OK!
Checking WebSocket directly to Tornado             OK!
Checking WebSocket through nginx                   OK!
Checking freeciv-web / publite2                    OK!
Checking webclient.min.js generation               OK!
Checking tileset generation                        OK!
Checking metaserver on PHP server                  OK!

--------------------------------
Check of Freeciv-web is done!
```

mchenryc committed:
Fix directory detection, normalize on bash

mchenryc committed:
Refactor status-freeciv-web.sh

* All checks give simple OK/Down status
* Check websockets by testing status of curl command
* Simplify process tests with reused functions
* Quiet output, and add a `-v` option for error messages

lonemadmax commented:
In status-freeciv-web.sh:
- Please review the use of `logfile`: in `checkURL` and `checkWS` it is read as a parameter but not used, whereas in `checkWebStatus` it is used without assignment.
- The webclient, tileset and metaserver checks lost the `--head` parameter to `curl`. Was that on purpose?
- If any of those are not right and you are changing this again, could you also remove the reference to PHP? It's been ages since the metaserver was rewritten.

mchenryc committed:
Remove unnecessary pw prompt, normalize on bash

mchenryc commented:
* logfile: thanks - forgot to remove logfile as a parameter to those functions after the last refactoring which changed logging so all tests (urls, websockets, pids, and services) use stderr/out in a common way from within `checkStatus` (curl sends info to stdout, when `-o logfile` is not used).
* added `--head` param back - using it saves bandwidth, though curl gives the same exit status regardless
* Ha! Yeah, actually I just removed the metaserver test entirely, given it's just another servlet in Tomcat. It gives us no more info about services being up/down than the "freeciv-web on Tomcat" test - LMK if you feel like it needs to be there

I also added local variable naming to a couple more functions, and made parameter order consistent, to make them easier to follow.

Lastly, a couple more start/stop scripts were changed to specify bash as well - see added commit.
